server/datastore/mysql/hosts.go

```diff
@@ -815,8 +832,23 @@ func (ds *Datastore) MarkHostsSeen(ctx context.Context, hostIDs []uint, t time.T
 func (ds *Datastore) SearchHosts(ctx context.Context, filter fleet.TeamFilter, matchQuery string, omit ...uint) ([]*fleet.Host, error) {
```

Comment:
`SearchHosts` also updated to mimic `ListHost` response

Reply:
> Do we need this here though? I think we only use this for target search.

Host search in the "Hosts" tab is performed by the UI?

Reply:
No, by filtering in ListHosts.

Reply:
Ah, your comment makes sense then (target search doesn't need this information).